repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -1016,6 +1020,13 @@ class SparkILoop(
     sparkContext
   }
 
+  @DeveloperApi
+  def createSQLContext(): SQLContext = {
+    sqlContext = new SQLContext(sparkContext)
```

Comment:
Can you use reflection here and if the `HiveContext` is available, use that one?